metadata/src/main/java/com/google/cloud/teleport/metadata/Template.java

```diff
@@ -112,4 +112,10 @@ enum TemplateType {
 
   /** Marker if the template is still in preview / pre-GA. */
   boolean preview() default false;
+
+  /**
+   * Comma-separated list of files to include in Template image when building with Dockerfile. Only
+   * works for YAML and XLANG types.
```

Comment:
Would be good to give an example - e.g. are these local files or can they be on gcs, etc...

Reply:
Actually, even better might be just giving the docker copy command here - that way its also clear where they're getting copied to